tests/tests_pytorch/test_cli.py

```diff
@@ -65,6 +68,22 @@ def mock_subclasses(baseclass, *subclasses):
         yield None
 
 
+@pytest.fixture
+def cleandir(tmp_path):
```

Comment:
perhaps we should use monkeypatch.chdir?


```py
@pytest.fixture
def cleandir(tmp_path, monkeypatch):
    monkeypatch.chdir(tmp_path)
    yield
```

?

Or how are we going to guarantee the correct working directory if one of these tests fails and doesn't change back? It would need a try: finally block at least right?

Reply:
Good idea!